src/core/components/fields/LabeledTextField.tsx

```diff
@@ -42,7 +40,7 @@ export const LabeledTextField = forwardRef<HTMLInputElement, LabeledTextFieldPro
           </div>
         )}
 
-        <style jsx>{`
+        <style jsx="true">{`
```

Comment:
Let's also check this because I believe I had take this stuff out to make it run.

Reply:
We need to assign the true to remove the following warning 
**Warning: Received `true` for a non-boolean attribute `jsx`.**
We encountered this first in PR306, but I am not sure if adding this assignment will create a problem with the application.